src/gui/virtualstudio.cpp

```diff
@@ -890,6 +896,7 @@ void VirtualStudio::completeConnection()
                 // run PLC without worker (4)
                 buffer_strategy = 4;
             }
+            */
         } else if (buffer_strategy == 5) {
             buffer_strategy = 3;  // run PLC with worker (3)
```

Comment:
Does this line also need to be changed to `buffer_strategy = 4;`?

Reply:
I don't think so. Leaving this as-is allows someone to still use the advanced setting to explicitly use a worker queue or not. It only changes the behavior of the "auto" setting so that it's the same as never use a worker.

I considered the alternative of just removing the option altogether so that there was only one PLC setting again, but I'd rather go slower and have more time to test in the wild before removing it entirely.

Also thinking about what happens for upgrade scenarios where the default value has always been 3 (after the +1):
* 3: PLC before auto -> PLC auto -> PLC with "no worker"
* 4: PLC after auto with "use worker" -> "use worker"
* 5: PLC after auto with "no worker" -> "no worker"

Maybe after next release I'd remove all the worker stuff entirely and force migrate everything to 3, but this approach allows us to still explicitly set to one or the other for a little while longer.